client/src/components/Navigation/NavigationItems/NavigationItems.jsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import classes from './NavigationItems.module.css';
 import NavigationItem from './NavigationItem/NavigationItem';
+import ProfileDropdown from '../../ProfileDropdown';
+import * as actions from '../../../store/actions/index';
 
-const navigationItems = () => (
+const navigationItems = ({ isAuthenticated, openAuthenticationModal }) => (
```

Comment:
You are creating re-usable presentational component right? Then why it has auth related logic and all the items are hard-coded. Please change the name of the component either to AuthNavigationItems or somewhat related to it.

Reply:
Actually initiallly I was passing these props  from app to all the way here then I realised I was beating the purpose of redux so connected this component to redux too.